pkgs/development/interpreters/erlang/generic-builder.nix

```
@@ -115,6 +117,9 @@ stdenv.mkDerivation ({
     patchShebangs make
 
     ${postPatch}
+  '' + optionalString patchShell ''
+    substituteInPlace lib/os_mon/src/disksup.erl \
+      --replace-fail '"sh ' '"${bash}/bin/sh '
```

```suggestion
      --replace-fail '"sh ' '"${runtimeShell} '
```
(Remember to add `runtimeShell` at top arguments too.)